api.raml

```diff
@@ -7,3 +7,9 @@ mediaType: application/json
     description: Delete file
     delete:
         description: Delete the matching file
+
+/delete_expiring_files/{exp_date}
```

Comment:
A couple of things:
  * it's missing a colon at the end of the line
  * usually in an API you use a single base, so I'll do something like

```
/delete:
  /file/{id}:
  /bulk/{exp_date}:
```